docs/source/notebooks/05_Advanced_RavenPy_configuration.ipynb

```diff
@@ -4,25 +4,20 @@
    "cell_type": "markdown",
```

Comment:
<p>review markup</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/424/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Same here, on the server it looks perfect. Perhaps a problem with ReviewNB?</p>